server/migrations/20230723093000_links.ts

```diff
@@ -0,0 +1,12 @@
+import { Knex } from "knex";
+
+export async function up(knex: Knex): Promise<any> {
+
+  await knex.schema.alterTable("links", table => {
+    table.specificType("target", "varchar").notNullable().alter();
```

Comment:
What did you get when altering to a string with no length?

Reply:
Why do you feel that's important? Especially seeing as we can keep it as the same type but simply remove the restriction. In doing so we keep it close to the original as possible, which is neater.

Reply:
Your example was missing a keyword and it was the wrong type, so I didn't try it

Reply:
Because I think it's much easier to review if the alter we perform here matches the schema builder calls in the initial table creation. I see I missed a bracket out, but I'm not sure what you mean about it being the wrong type?